tools/sgjsonfmt/tools.go

```diff
@@ -0,0 +1,56 @@
+package sgjsonfmt
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const (
+	version    = "0.5.0"
+	binaryName = "jsonfmt"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir("jsonfmt"), args...)
```

Comment:
```suggestion
	return sg.Command(ctx, sg.FromBinDir(binaryName), args...)
```

Perhaps?

Reply:
I am for it. This is my first Go PR. I just copied the copy-paste-style of https://github.com/einride/sage/blob/master/tools/sgyamlfmt/tools.go.